### DISCUSSION TITLE
Возможно ли реализовать "Городской портал"?

### DISCUSSION THREAD

**Discussion by KsenZ** [maxsite/cms#434] [Q&A] [answered]
Достался мне сайт на Joomla с неким шаблоном "Городской портал".
Демку этого портала портала можно посмотреть на [https://gorod1.pro-portal.info/](url)
Не нравится он мне по многим причинам. Наверное основные из них, это отсутствие внятной документации и невозможность задать вопрос на форуме, т.к. общаться можно если только есть активная подписка.
Да и в целом "тормознутость" этой поделки. Там где "разжиревшая" Bitrix CMS, работает вполне отзывчиво, сайт с этим шаблоном "тормозит", хотя на нем еще и контента толком нет.

Собственно сабж.
Возможно ли, что то подобное реализовать на Maxsite CMS? Может быть уже что то реализовано с шаблоне MF?

Сразу хочу сказать, что львиная доля функционала, представленного в том шаблоне, не нужна.
Нужен, например, каталог организаций города [https://gorod1.pro-portal.info/katalog](url), и "Народная карта" [https://gorod1.pro-portal.info/narodnaya-karta](url)

Или нужны будут большие переделки Maxsite CMS, которые не стоит делать из-за одного сайта?

**Comment by maxsite** [comment, marked as answer]
Для такого сайта нужно хорошо продумать структуру хранения данных. Есть итоговая запись (page), есть группировка по рубрикам (category) и есть метки (tag). Если вы уложитесь в эти сущности, то это сильно упрощает задачу. Если этого не хватает, то вводятся метаполя записи и какие-то новые типы данных (type-файлы), через которые осуществляется вывод page (в одиночном виде или в виде групп/архива). 

Для этого не нужные никакие переделки MaxSite CMS, поскольку весь функционал находится на уровне шаблона.

**Reply by KsenZ** [reply]
> весь функционал находится на уровне шаблона.

В данном, конкретном случае, есть смысл в MF шаблоне?

**Reply by maxsite** [reply]
В MF много разных блоков, но вам придётся всё равно делать свой дизайн, поэтому MF вряд ли подойдёт. Лучше взять Default-шаблон, поскольку он простой и «чистый» и уже на его основе что-то делать.
